CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+**JDBC Driver 3.13.28**
+
+- \|| Please Refer to Release Notes at https://community.snowflake.com/s/article/JDBC-Driver-Release-Notes
```

Comment:
If it's the same link, do we have to keep adding for each release?

Reply:
Agree, should we even be mainitaing the change log list